### PR TITLE
Fixes developed from mypy_django script

### PR DESCRIPTION
**Pull request by tabletcorry**
A few discoveries from running mypy with Django-awareness. Nothing critical, but it caught a few odds and ends.

### CODE REVIEW

File: core/files.py [hunk 1]
```diff
@@ -2,19 +2,24 @@
 
 import blurhash
 import httpx
+import PIL.ImageQt
```
Comment (andrewgodwin) [inline core/files.py:5]: Wait what is this doing here, since it's not referenced elsewhere?

Comment (tabletcorry) [reply core/files.py:5]: Uh, good question. I have no idea.

Comment (tabletcorry) [reply core/files.py]: I think something misfired when I typed it, and imported the reference. Anyways, fixed now.

**Review by andrewgodwin — APPROVED** [review]

**Comment by andrewgodwin** [comment]
Lovely, thanks!

**Pull request merged**